src/rust/generators/sysmon-generator/tests/integration_test.rs

```diff
@@ -145,7 +145,7 @@ async fn test_sysmon_event_produces_expected_graph(ctx: &mut SysmonGeneratorTest
     let kafka_subscriber = tokio::task::spawn(async move {
         let stream = kafka_consumer
             .stream()
-            .expect("could not subscribe to the generated-graphs topic");
+            .expect("could not subscribe to the topic");
```

Comment:
what was the rationale behind this change?

Reply:
Honest answer: much of this code is copy-pasted, and this mid-string thing is hard to catch. In fact, back when I thought this integration test was also going to be a Kafka-based integration test, I'd missed this during copy-paste. I figured that it was simple enough to look and see "oh, the topic is CONSUMER_TOPIC=." 

I really wish this could be a `format!("...{}...", CONSUMER_TOPIC)`, but it has to be an &'static str.

I'm not completely attached to this change, I can revert if you'd like.

Reply:
Honest answer: much of this code is copy-pasted, and this mid-string thing is hard to catch. In fact, back when I thought this integration test was also going to be a Kafka-based integration test, I'd missed this during copy-paste. I figured that it was simple enough to look and see "oh, the topic is CONSUMER_TOPIC=." 

I really wish this could be a `format!("...{}...", CONSUMER_TOPIC)`, but it has to be an &'static str.

I'm not completely attached to this change, I can revert if you'd like.